ntbs-service/Services/UserService.cs

```diff
@@ -53,18 +54,27 @@ public async Task<TBService> GetDefaultTBService(ClaimsPrincipal user)
             var type = GetUserType(user);
             var roles = GetRoles(user);
 
-            if (type == UserType.NationalTeam)
-            {
-                return null;
-            }
+            return await UserServicesQuery(type, roles).FirstAsync();
+        }
 
-            if (type == UserType.NhsUser)
-            {
-                return await context.TbService.FirstAsync(tb => roles.Contains(tb.ServiceAdGroup));
-            }
-            else
+        public async Task<IEnumerable<TBService>> GetTbServicesAsync(ClaimsPrincipal user)
+        {
+            var type = GetUserType(user);
+            var roles = GetRoles(user);
+
+            return await UserServicesQuery(type, roles).ToListAsync();
+        }
+
+        private IQueryable<TBService> UserServicesQuery(UserType type, IEnumerable<string> roles)
```

Comment:
Could we rename this to GetTBServiceQueryable or similar, to make clear this is specific to tbservices?